### DISCUSSION TITLE
Avoid error when stylesheet missing for a pack

### DISCUSSION THREAD

**Discussion by devsheva** [shakacode/shakapacker#527] [Q&A] [answered]
Before migrating from webpacker to shakapacker, no errors where raised when a stylesheet was missing for relative pack.

Like i have a pack called core.js, the precompile expects a core.css. Is this an intended behavior? Cause i can't find anything about it.

**Comment by justin808** [comment]
Can you create a small test repository to show the issue? Your question is too short for anybody to help you. Given the amount of time that Shakapacker has been out, there's probably something specific about your setup.

**Reply by devsheva** [reply]
Sorry for the late answer, sure I'll do it. Do you have a reproducible template?

**Reply by tomdracz** [reply]
Don't think we do currently, but you could start from something like https://github.com/tomdracz/shakapacker-css-test That I've been using as a quick scaffold to debug issues coming up

**Reply by devsheva** [reply]
Here you go. https://github.com/devsheva/shakapacker-css-test
Just run `rails assets:precompile` and then `rails s`

**Reply by Judahmeek** [reply, marked as answer]
@devsheva I'm sorry that we did not respond in a timely manner.

The `stylesheet_pack_tag` is what is requiring `core.css`, so just remove that.

Although usually, you'll have one `stylesheet_pack_tag` that doesn't call anything and multiple `append_stylesheet_pack_tag`, each calling a different stylesheet. In that case, only have `append_stylesheet_pack_tag` calls for stylesheets that actually exist.
